src/services/polling/modbus_polling.py

```diff
@@ -126,6 +126,20 @@ def __poll_network_devices(self, current_connection, network: NetworkModel):
                     else:
                         raise Exception(f'FC {point.function_code} unsupported for aggregate')
 
+                def is_point_to_be_written(point: PointModel) -> bool:
+                    write_value: float = PriorityArrayModel.get_highest_priority_value_from_priority_array(
+                        point.priority_array_write)
+                    return not((point.is_writable(point.function_code) and write_value is None) or (
+                        point.is_writable(point.function_code) and point.write_value_once and
+                        point.point_store is not None and not point.point_store.fault and
+                        point.point_store.value_original == write_value))
+
+                def clear_write_points(point_list: List[PointModel]):
+                    point_list[:] = [p for p in point_list if is_point_to_be_written(p)]
+
+                clear_write_points(fc_lists[4])
+                clear_write_points(fc_lists[5])
```

Comment:
This would be better:

```
                self.clear_write_points(fc_lists[4])
                self.clear_write_points(fc_lists[5])
```

on here, and move the block outside of this method scope (inner function is just confusing sometimes)

```
    @staticmethod
    def is_point_to_be_written(point: PointModel) -> bool:
        write_value: float = PriorityArrayModel.get_highest_priority_value_from_priority_array(
            point.priority_array_write)
        return not ((point.is_writable(point.function_code) and write_value is None) or (
                point.is_writable(point.function_code) and point.write_value_once and
                point.point_store is not None and not point.point_store.fault and
                point.point_store.value_original == write_value))

    @staticmethod
    def clear_write_points(point_list: List[PointModel]):
        point_list[:] = [p for p in point_list if ModbusPolling.is_point_to_be_written(p)]

```